src/navigation/context.py

```diff
@@ -13,7 +13,17 @@
 from mrover.msg import Waypoint, GPSWaypoint, EnableAuton
 import pymap3d
 from std_msgs.msg import Time
+import yaml
 
+   
+
+
+# read required parameters, if they don't exist an error will be thrown
+REF_LAT = rospy.get_param("gps_linearization/reference_point_latitude")
+REF_LON = rospy.get_param("gps_linearization/reference_point_longitude")
+
+MAP = rospy.get_param("gps_linearization/world_frame")
```

Comment:
change these constants from "MAP" to "WORLD_FRAME" and "BASE_LINK" to "ROVER_FRAME" that way they're more general